remail/controller.py

```
@@ -111,27 +112,31 @@
+        try:
+            with Session(self.engine) as session:
+                existing_contact = session.exec(select(Contact).where(Contact.email_address == email_address)).first()
+                if existing_contact:
+                    raise ValueError(f"Kontakt mit der E-Mail-Adresse {email_address} existiert bereits.")
+
+                contact = Contact(email_address=email_address, name=name)
+                session.add(contact)
+                session.commit()
+                return contact
+        except Exception as e:
+            raise RuntimeError(f"Fehler beim Erstellen des Kontakts: {str(e)}") 
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)